### ISSUE TITLE
NotoSansGujarati-Regular 1.0.3 fails for specific sequence

### ISSUE THREAD

**Issue by blagae** [googlefonts/noto-fonts#1272]
For the character sequence, NotoSansGujarati-Regular 1.0.3 displays wrong result for `\u0A87\u0AA6\u0ACD\u0AA6\u0ACD\u0AB0\u0ABF\u0AB6`

Result for Noto in Notepad++ (unacceptable):

![image](https://user-images.githubusercontent.com/8480167/44664124-a9d14e80-aa12-11e8-8d86-945baecbea59.png)

Result for Arial Unicode in Notepad++ (acceptable):

![image](https://user-images.githubusercontent.com/8480167/44664183-ca99a400-aa12-11e8-8e28-fa7264e0f31f.png)

Result for Shruti in Notepad++ (good):

![image](https://user-images.githubusercontent.com/8480167/44664208-dd13dd80-aa12-11e8-9b86-a61b3e921669.png)

Either of the last two would be (close enough to) an expected result, even though the last one is much more idiomatic. The biggest problem is that the `\u0ABF` glyph is not repositioned correctly. A lesser problem (as in, the word is still readable) is the lack of glyph replacement in the sequence `\u0AA6\u0ACD\u0AA6\u0ACD\u0AB0`

**Comment by JelleBosmaMT** [comment]
I cannot find words with d-d-ra, and a websearch with ઇદ્દ્રિશ gives no results. The fact that I cannot find something does not proof it does not exist. But it suggest it is not used very much. In what context is this ligature needed?

**Comment by blagae** [comment]
I'm no longer on the project that I was on when I created this issue. I have no access anymore to the accounts that I used there, and when I asked former co-workers, they haven't been able to find anything related to this. I'm also not an expert on Gujarati myself. So in short: I can't remember.

I do believe that the issue came from a user asking why there was a difference between the Shruti font and Noto; this is the closest I can get to guessing its relevance. Since there is a difference with other fonts, and since it probably came from a user, I have the impression that it is a valid cluster, but I trust your judgement on whether or not you want to fix this (i.e. I have no personal interest in the fix anymore).
